src/index.ts

```diff
@@ -1,38 +1,50 @@
 import type { Server } from 'http';
 import { createServer as createHttpServer } from 'http';
 import { ApolloServer } from 'apollo-server-express';
-import { applyMiddleware } from 'graphql-middleware';
 import type express from 'express';
-import { PORT, isDevelopment } from './env';
+import type { Disposable } from 'graphql-ws';
+import { ApolloServerPluginDrainHttpServer } from 'apollo-server-core';
 import { createApp } from './app';
+import { PORT, isDevelopment } from './env';
 import { createContext } from './context';
-import { schema } from './schema';
+import { schemaWithMiddleware } from './schema';
 
-export const schemaWithMiddleware = applyMiddleware(schema);
+let serverCleanup: Disposable;
```

Comment:
You never actually assign anything to this variable

Reply:
removed whatsovever.